cli/dcoscli/package/main.py

```diff
@@ -14,18 +15,24 @@
     dcos package update [--validate]
 
 Options:
-    -h, --help         Show this screen
-    --info             Show a short description of this subcommand
-    --version          Show version
-    --yes              Assume "yes" is the answer to all prompts and run
-                       non-interactively
-    --all              Apply the operation to all matching packages
-    --app              Apply the operation only to the package's application
-    --app-id=<app-id>  The application id
-    --cli              Apply the operation only to the package's CLI
-    --options=<file>   Path to a JSON file containing package installation
-                       options
-    --validate         Validate package content when updating sources
+    -h, --help                          Show this screen
+    --info                              Show a short description of this
+                                        subcommand
+    --version                           Show version
+    --yes                               Assume "yes" is the answer to all
+                                        prompts and run non-interactively
+    --all                               Apply the operation to all matching
+                                        packages
+    --app                               Apply the operation only to the
+                                        package's application
+    --app-id=<app-id>                   The application id
+    --cli                               Apply the operation only to the
+                                        package's CLI
+    --options=<file>                    Path to a JSON file containing package
+                                        installation options
+    --validate                          Validate package content when updating
+                                        sources
+    --package-version=<package_version> Package version to install
```

Comment:
I say we move these to data files, and standardize on keeping the doc part (not the option part) to 70-char width for readability.